### PR TITLE
Merge ProjectsPage and ProjectPage

### LINKED ISSUE #1602: Use PageTitlePreset on all pages
- refactor props, ex: user can be user from pageContext
- projects page
- project page

### LINKED ISSUE #1603: Merge ProjectsPage and ProjectPage
Current diff:
- title
- projects count
- PageActions

### PR DESCRIPTION
**Pull request by DenisVorop**
## PR includes
- [x] Refactor

## Related issues
Resolve #1603 
Resolve #1602 

## QA Instructions, Screenshots, Recordings

https://github.com/taskany-inc/issues/assets/95316053/6dc573bb-109c-4f19-89ea-8204a8a46b68

### CODE REVIEW

**Comment by 9teen90nine** [comment]
what about project page?

File: src/components/ProjectsPage/ProjectsPage.tsx [hunk 1]
```diff
@@ -106,6 +134,22 @@ export const ProjectsPage = ({ user, ssrTime, defaultPresetFallback }: ExternalP
...
+    useEffect(() => {
+        if (project) {
+            setCurrentProjectCache({
+                id: project.id,
+                title: project.title,
+                description: project.description ?? undefined,
+                flowId: project.flowId,
+            });
+        }
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (9teen90nine) [inline src/components/ProjectsPage/ProjectsPage.tsx:146]: we don't do that

File: src/hooks/useProjectResource.ts [hunk 2]
```diff
@@ -92,6 +100,8 @@ export const useProjectResource = (id: string) => {
 
     const transferOwnership = useCallback(
         (cb: Callback, activityId: string) => async () => {
+            if (!id) return;
```
Comment (9teen90nine) [inline src/hooks/useProjectResource.ts:103]: maybe you should create a guard function to avoid this copypaste?

**Review by awinogradov — CHANGES_REQUESTED** [review]
If we have `ProjectsPage` only do we really need `ProjectPageLayout`?

**Review by 9teen90nine — APPROVED** [review]

**Pull request merged**